components/FriendCodesPanel.tsx

```diff
@@ -72,54 +75,54 @@ export default function FriendCodesPanel() {
     }, []);
 
     return (
-        <header className={FormStyles.header}>
-            <Forms.FormTitle
+        <header className={FormStyles.header} style={{ display: "block" }}>
```

Comment:
It often gets the wrong FormStyles due to the Filter. Please add the Padding so that it looks right in case it gets the wrong class
```suggestion
        <header className={FormStyles.header} style={{ display: "block", padding: "20px 30px" }}>
```